src/shared/Utils.cs

```diff
@@ -1,11 +1,20 @@
+using System.Text;
 using System.Text.Json;
 using System.Text.RegularExpressions;
 using System.Dynamic;
+using System.Security.Cryptography;
 
 namespace dig;
 
 internal static partial class Utils
 {
+    public static string MD5Hash(this string input)
```

Comment:
probably doesnt matter but i was thinking this morning if it made more sense to do sha256. The only advantage is less chance of collision.....but chance of collision is probably super low so might not need to worry about it.

Reply:
the only reason id pick md5 is for speed. since this isn't a security sensitive hash, md5 is fine.

https://debugpointer.com/security/md5-vs-sha256